twill-core/src/main/java/org/apache/twill/internal/state/SystemMessages.java

```diff
@@ -68,6 +73,21 @@ public static Message updateRunnablesInstances(Command updateCommand) {
     return new SimpleMessage(Message.Type.SYSTEM, Message.Scope.RUNNABLES, null, updateCommand);
   }
 
+  public static Message setLogLevel(Map<String, LogEntry.Level> logLevelArguments) {
+    return setLogLevel(Constants.SystemMessages.LOG_ALL_RUNNABLES, logLevelArguments);
```

Comment:
The constants about log levels are only used for `Messages`, they are also used in `TwillPreparer`.